tests/test_urlparse.py

```diff
@@ -150,6 +150,26 @@ def test_param_with_existing_escape_requires_encoding():
     assert str(url) == "http://webservice?u=http%3A%2F%2Fexample.com%3Fq%3Dfoo%252Fa"
 
 
+def test_param_with_existing_escape():
+    url = httpx.URL("https://webservice/?u=/%3D%26&v=1%202")
+    assert str(url) == "https://webservice/?u=%2F%3D%26&v=1%202"
```

Comment:
I don't think this will work, it's still corrupting the parameter by encoding `/` to `%2F` which is something not all servers can handle.